com.musala.xml.parser.executable/src/main/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+		<groupId>com.musala.data.parser</groupId>
+		<artifactId>com.musala.xml.parser.executable.modules</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+    </parent>
+  
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.musala.data.parser</groupId>
+	<artifactId>com.musala.xml.parser.executable.src</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<build>
+		<sourceDirectory>../../src/main/java</sourceDirectory>
+		<plugins>
```

Comment:
Add here the buildnumber-maven-plugin (present in the top-most pom of the project in this commit https://github.com/StefanSimeonov/com.musala.data.parser/commit/2687819438e298f180b0e8c238938ce0a9719060) so that the source generated jar has a timestamp.